02-intermediate-ruby/api-testing-vcr.md

```diff
@@ -20,6 +20,23 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
 
 ![VCR](images/vcr.jpg)
 
+### What is VCR doing?
+VCR is essentially a recording device that interrupts the normal way they we send messages out to the internet. The first time a set of tests that use VCR run, the cassette takes note of the new URL:
```

Comment:
Suggestion: the cassette takes note of the new ~URL~ request, based off of request method, URI, and body: